src/pages/pool/withdraw.vue

```diff
@@ -30,15 +31,25 @@ const { activeTab, resetTabs } = useWithdrawPageTabs();
 // onchain calls. i.e. only refetch what's required to be up to date for joins/exits.
 useIntervalFn(poolQuery.refetch.value, oneMinInMs);
 
+/**
+ * COMPUTED
+ */
+// We only need to wait for SOR if it's a deep pool.
+const isLoadingSor = computed(
+  (): boolean => isDeepPool.value && hasFetchedPoolsForSor.value
```

Comment:
Should be `isDeepPool.value && !hasFetchedPoolsForSor.value`? Other than that, lgtm.

Reply:
Correct, good catch, thanks.